[2023-05-04 05:06:53,887] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.extract_from_s3 manual__2023-05-03T18:42:51.223967+00:00 [queued]>
[2023-05-04 05:06:54,158] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag_etl.extract_from_s3 manual__2023-05-03T18:42:51.223967+00:00 [queued]>
[2023-05-04 05:06:54,168] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-05-04 05:06:54,170] {taskinstance.py:1357} INFO - Starting attempt 1 of 6
[2023-05-04 05:06:54,171] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-05-04 05:06:54,523] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): extract_from_s3> on 2023-05-03 18:42:51.223967+00:00
[2023-05-04 05:06:54,552] {standard_task_runner.py:52} INFO - Started process 329 to run task
[2023-05-04 05:06:54,587] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag_etl', 'extract_from_s3', 'manual__2023-05-03T18:42:51.223967+00:00', '--job-id', '906', '--raw', '--subdir', 'DAGS_FOLDER/dag_etl_taskflow.py', '--cfg-path', '/tmp/tmpen5xeo9p', '--error-file', '/tmp/tmph3o7s_5e']
[2023-05-04 05:06:54,592] {standard_task_runner.py:80} INFO - Job 906: Subtask extract_from_s3
[2023-05-04 05:06:55,723] {task_command.py:370} INFO - Running <TaskInstance: dag_etl.extract_from_s3 manual__2023-05-03T18:42:51.223967+00:00 [running]> on host 295d97741f96
[2023-05-04 05:06:58,555] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Victor
AIRFLOW_CTX_DAG_ID=dag_etl
AIRFLOW_CTX_TASK_ID=extract_from_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-05-03T18:42:51.223967+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-03T18:42:51.223967+00:00
[2023-05-04 05:06:58,574] {s3.py:861} INFO - Downloading source S3 file from Bucket raw with path supermarket_sales.csv
[2023-05-04 05:06:59,170] {base.py:68} INFO - Using connection ID 'minio_conn' for task execution.
[2023-05-04 05:06:59,179] {base_aws.py:206} INFO - Credentials retrieved from login
[2023-05-04 05:07:03,955] {python.py:173} INFO - Done. Returned value was: /opt/***/data/***_tmp_nfp78dr7
[2023-05-04 05:07:04,701] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_etl, task_id=extract_from_s3, execution_date=20230503T184251, start_date=20230504T050653, end_date=20230504T050704
[2023-05-04 05:07:05,104] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-05-04 05:07:06,660] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
